README.md

```diff
@@ -0,0 +1,51 @@
+# Amity Room Allocation
+
+[![Build Status](https://travis-ci.org/ivantoz/Office-Space-Allocation.svg?branch=dev)](https://travis-ci.org/ivantoz/Office-Space-Allocation)
+[![Coverage Status](https://coveralls.io/repos/github/ivantoz/Office-Space-Allocation/badge.svg?branch=dev)](https://coveralls.io/github/ivantoz/Office-Space-Allocation?branch=dev)
+
+Amity is a room allocation and management system for one of andela's facilities called 'Amity'
+
+**Installation**
+
+`$ git clone -b dev https://github.com/ivantoz/Office-Space-Allocation.git`
+
+`$ cd amity/`
+
+Create and activate a virtual environment
+
+```
+
+$ virtualenv -p /usr/bin/python3 .env
+$ source .env/bin/activate
+
+```
+
+Install dependencies
+`$ pip install -r requirements.txt`
+
+**Run the application**
+```
+
+$ python app.py -i
+
+```
+
```

Comment:
you can improve on your README by showing output from running the program such as showing command and output from the program